lms/djangoapps/grades/tasks.py

```diff
@@ -189,7 +192,14 @@ def _update_subsection_grades(
                         subsection_grade=subsection_grade,
                     )
 
-        except DatabaseError as exc:
+        except Exception as exc:   # pylint: disable=broad-except
+            if not isinstance(exc, KNOWN_RETRY_ERRORS):
+                log.info("tnl-6244 grades unexpected failure: {}. user_id, usage_id, course_id: {}, {}, {}".format(
```

Comment:
To help us debug this issue, can you output all the kwargs passed to the task?
In order to do this, you may need to move the try-except to the caller of this function.
Also, I just noticed that this try-except would not have caught the ORA issue since it's thrown before this function is even called.  So I suggest we have the try-except surround the entire code within the task function.